apps/passport-client/components/screens/FrogScreens/FrogHomeScreen.tsx

```diff
@@ -11,60 +19,31 @@ import { useSyncE2EEStorage } from "../../../src/useSyncE2EEStorage";
 import { MaybeModal } from "../../modals/Modal";
 import { AppContainer } from "../../shared/AppContainer";
 import { AppHeader } from "../../shared/AppHeader";
-import { PCDCard } from "../../shared/PCDCard";
 import { SyncingPCDs } from "../../shared/SyncingPCDs";
+import { ActionButton, Button } from "./Button";
+import { DexTab } from "./DEXTab";
```

Comment:
<img width="921" alt="Screenshot 2023-10-31 at 12 01 24 PM" src="https://github.com/proofcarryingdata/zupass/assets/2636237/ded6c85c-321b-4ce5-97a5-348183e49c2d">

getting this locally - be careful about file name capitalization. this can sometimes lead to bugs. eg. if you import the same module and reference it w/ two different capitalizations, node will treat that as two separate modules. this can lead to unexpected behavior, eg. if the module contains a global variable, you'll have two divergent global states.

Reply:
:( Oh yes. I thought I fixed this. I named it DEX and later changed my mind. Fixed now